client/rpcserver/handlers.go

```diff
@@ -691,6 +698,66 @@ func handleDeleteArchivedRecords(s *RPCServer, params *RawParams) *msgjson.Respo
 	return createResponse(deleteArchivedRecordsRoute, msg, nil)
 }
 
+func handleWalletPeers(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
+	if len(params.Args) != 1 {
+		return usage(walletPeersRoute, errors.New("expected 1 arg"))
+	}
+	assetID, err := checkUIntArg(params.Args[0], "assetID", 32)
+	if err != nil {
+		return usage(walletPeersRoute, err)
+	}
+
+	peers, err := s.core.WalletPeers(uint32(assetID))
+	if err != nil {
+		errMsg := fmt.Sprintf("unable to get wallet peers: %v", err)
+		resErr := msgjson.NewError(msgjson.RPCWalletPeersError, errMsg)
+		return createResponse(walletPeersRoute, nil, resErr)
+	}
+	return createResponse(walletPeersRoute, peers, nil)
+}
+
+func handleAddWalletPeer(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
+	if len(params.Args) != 2 {
+		return usage(walletPeersRoute, errors.New("expected 2 args"))
+	}
+	assetID, err := checkUIntArg(params.Args[0], "assetID", 32)
+	if err != nil {
+		return usage(walletPeersRoute, err)
+	}
+
+	addr := params.Args[1]
+
+	err = s.core.AddWalletPeer(uint32(assetID), addr)
+	if err != nil {
+		errMsg := fmt.Sprintf("unable to get wallet peers: %v", err)
+		resErr := msgjson.NewError(msgjson.RPCWalletPeersError, errMsg)
+		return createResponse(walletPeersRoute, nil, resErr)
+	}
+
+	return createResponse(walletPeersRoute, "successfully added peer", nil)
+}
+
+func handleRemoveWalletPeer(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
+	if len(params.Args) != 2 {
+		return usage(walletPeersRoute, errors.New("expected 2 args"))
+	}
+	assetID, err := checkUIntArg(params.Args[0], "assetID", 32)
+	if err != nil {
+		return usage(walletPeersRoute, err)
+	}
+
+	addr := params.Args[1]
+
+	err = s.core.RemoveWalletPeer(uint32(assetID), addr)
+	if err != nil {
+		errMsg := fmt.Sprintf("unable to get wallet peers: %v", err)
```

Comment:
The message applies to `handleWalletPeers`.  All the route constants are `walletPeersRoute`, but I think you want `addWalletPeerRoute` or `removeWalletPeerRoute` for these to latter handlers.